app/src/app/navtitle/navtitle.component.sass

```diff
@@ -63,3 +79,11 @@ nav
     background-color: color.adjust($maincolor, $lightness: 10%)
   a.active:hover
     background-color: $nav-hover-bg
+
+hr
+  background-color: $maincolor
+  border: none
+  height: 1px
+  width: 120px
```

Comment:
This is not solid: if we ass an item with a long name in the menu, we'll have to change this length. Use percent as unit.